src/libraries/System.Private.Uri/src/System/UriBuilder.cs

```diff
@@ -2,26 +2,23 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Diagnostics.CodeAnalysis;
+using System.Text;
 
 namespace System
 {
     public class UriBuilder
     {
-        // fields
-
-        private bool _changed = true;
-        private string _fragment = string.Empty;
-        private string _host = "localhost";
+        private string _scheme = "http";
+        private string _username = string.Empty;
         private string _password = string.Empty;
-        private string _path = "/";
+        private string _host = "localhost";
         private int _port = -1;
+        private string _path = "/";
         private string _query = string.Empty;
-        private string _scheme = "http";
-        private string _schemeDelimiter = Uri.SchemeDelimiter;
-        private Uri _uri = null!; // initialized in ctor via helper
-        private string _username = string.Empty;
+        private string _fragment = string.Empty;
 
-        // constructors
+        private bool _changed = true;
+        private Uri _uri = null!; // initialized in ctor via helper
```

Comment:
```suggestion
        private Uri _uri;
```

Reply:
Now the field is no longer initialized in `Init`, but directly in the constructor, this is not necessary.

Reply:
It should actually be a nullable field since not all ctor overloads set this - the comment is also sometimes wrong, I'll remove it.